include/cadmium/modeling/dynamic_model.hpp

```diff
@@ -28,18 +28,19 @@
 #define CADMIUM_ATOMIC_HPP
 
 #include <cadmium/modeling/dynamic_message_bag.hpp>
+#include <cadmium/engine/pdevs_dynamic_link.hpp>
 
 namespace cadmium {
     namespace dynamic {
         namespace modeling {
 
             struct EOC {
                 std::string _from;
-                std::shared_ptr<cadmium::dynamic::link_abstract> _link;
+                std::shared_ptr<cadmium::dynamic::engine::link_abstract> _link;
```

Comment:
I'm not convinced yet that engine is the right place for the link, since we are using it for modeling purposes. But, it is ok to keep it here for now until we figure out what would be a better place.

Reply:
We are using it for both, modeling and simulation purposes, the model uses this class to defined the couplings as you have pointed out here, but the engine uses it for routing the messages.

A link has the power to take two cadmium::dynamic::message_bags and rout the messages from the PORT_FROM of the first dynamic bag to the PORT_TO of the second bag.

I think it could go in a new folder called "simulable_modeling" or just "simulable". Are you agree?